### ISSUE TITLE
can't start the project 

### ISSUE THREAD

**Issue by ZaheerAbbasAghani** [akaunting/akaunting#204]
Warning: Unknown: failed to open stream: No such file or directory in Unknown on line 0

Fatal error: Unknown: Failed opening required 'F:\laragon\www\akaunting-master/server.php' (include_path='.;F:/laragon/etc/php/pear') in Unknown on line 0

**Comment by sebadamus** [comment]
Hi, what your environment? Windows, right? which php/apache package are you using? WAMP, XAMPP?

You download the project, right? and after that have you follow the compilation instructions?

**Comment by ZaheerAbbasAghani** [comment]
window and laragon. laragon has both php/apache and xampp and i followed below instructions.

Install Composer
Download the repository and unzip into your server
Open and point your command line to the directory you unzipped Akaunting
Run the following command: composer install
Finally, launch the installer

**Comment by sebadamus** [comment]
And "composer install" did not gave you any error or warning? because I installed the source and followed instructions there and worked just fine (I remember some ubuntu packages I had to install before the "composer install" command, but just figured out in the process by reading the errors)

I test it only on Ubuntu.  On Windows  I just download the package from the main site akaunting web https://akaunting.com/download (its not only not the source, you have not to "composer install" first), may be it works you right away.
